.github/workflows/review.yml

```diff
@@ -0,0 +1,26 @@
+name: review
+ 
+on:
+  pull_request:
+    types: [opened, synchronize]
+ 
+jobs:
+  review:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+ 
+    steps:
+      - name: checkout 
+        uses: actions/checkout@v1
+ 
+      - name: project-version-check (integration-test)
+        uses: ./
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          file-to-check: package.json
+          additional-files-to-check: README.md
+
+      - name: test
+        run: |
+          npm install
+          npm test
```

Comment:
Evtl. die UT vor dem IT?

Reply:
Ohne den `install`, funktioniert der `test` nicht.